src/javascript/app/base/header.js

```diff
@@ -275,43 +275,47 @@ const Header = (() => {
+                document_expired      : () => buildMessage(localize('Your proof-of-identity documents have expired and your account has been limited. Please send updated documents to customer support.')),
+                document_expiring_soon: () => buildMessage(localize('Your proof-of-identity documents will expire within the next month. Please send updated documents to customer support to prevent your account being limited.')),
+                excluded_until        : () => buildMessage(localizeKeepPlaceholders('Your account is restricted. Kindly [_1]contact customer support[_2] for assistance.'),                                                             'contact'),
+                financial_limit       : () => buildMessage(localizeKeepPlaceholders('Please set your [_1]30-day turnover limit[_2] to remove deposit limits.'),                                                                         'user/security/self_exclusionws'),
+                mf_retail             : () => buildMessage(localizeKeepPlaceholders('Binary Options Trading has been disabled on your account. Kindly [_1]contact customer support[_2] for assistance.'),                               'contact'),
+                mt5_withdrawal_locked : () => localize('MT5 withdrawals have been disabled on your account. Please check your email for more details.'),
+                required_fields       : () => buildMessage(localizeKeepPlaceholders('Please complete your [_1]personal details[_2] before you proceed.'),                                                                               'user/settings/detailsws'),
+                residence             : () => buildMessage(localizeKeepPlaceholders('Please set [_1]country of residence[_2] before upgrading to a real-money account.'),                                                               'user/settings/detailsws'),
+                risk                  : () => buildMessage(localizeKeepPlaceholders('Please complete the [_1]financial assessment form[_2] to lift your withdrawal and trading limits.'),                                               'user/settings/assessmentws'),
+                tax                   : () => buildMessage(localizeKeepPlaceholders('Please [_1]complete your account profile[_2] to lift your withdrawal and trading limits.'),                                                        'user/settings/detailsws'),
+                unwelcome             : () => buildMessage(localizeKeepPlaceholders('Trading and deposits have been disabled on your account. Kindly [_1]contact customer support[_2] for assistance.'),                                'contact'),
+                withdrawal_locked     : () => localize('Withdrawals have been disabled on your account. Please check your email for more details.'),
+                tnc                   : () => buildMessage(has_no_tnc_limit
```

Comment:
the reason why tnc is not alphabetically sorted is that it has multiple lines, so we kept it at the bottom separately